src/ray/object_manager/test/get_request_queue_test.cc

```diff
@@ -106,7 +108,9 @@ TEST_F(GetRequestQueueTest, TestObjectSealed) {
   GetRequestQueue get_request_queue(
       io_context_,
       object_lifecycle_manager,
-      [&](const ObjectID &object_id, const auto &request) {},
+      [&](const ObjectID &object_id,
+          std::optional<MEMFD_TYPE> fallback_allocated_fd,
```

Comment:
can you add a simple unit test to verify this is passed correctly?

Reply:
done.